internal/controllers/import_controller.go

```diff
@@ -50,10 +48,11 @@ import (
 	"sigs.k8s.io/cluster-api/util/predicates"
 	utilyaml "sigs.k8s.io/cluster-api/util/yaml"
 
-	"github.com/rancher-sandbox/rancher-turtles/internal/rancher"
+	managementv3 "github.com/rancher-sandbox/rancher-turtles/internal/rancher/management/v3"
+	provisioningv1 "github.com/rancher-sandbox/rancher-turtles/internal/rancher/provisioning/v1"
 	"github.com/rancher-sandbox/rancher-turtles/util"
 	turtlesannotations "github.com/rancher-sandbox/rancher-turtles/util/annotations"
-	turtelesnaming "github.com/rancher-sandbox/rancher-turtles/util/naming"
+	turtlesnaming "github.com/rancher-sandbox/rancher-turtles/util/naming"
```

Comment:
Can we add these 3 to the https://github.com/rancher-sandbox/rancher-turtles/blob/main/.golangci.yml#L41-L62 so that we use the consistent imports in the future?